File: contests/abc122/abc122_b.py
# B - ATCoder
# https://atcoder.jp/contests/abc122/tasks/abc122_b

S = input()
ans = 0
count = 0
 
for char in S:
    if char in 'ACGT':
        count += 1
    else:
        count = 0
    ans = max(ans, count)
print(ans)

File: contests/abc122/abc122_b.cpp
// B - ATCoder
// https://atcoder.jp/contests/abc122/tasks/abc122_b

#include <iostream>
#include <string>
using namespace std;

int main() {
    string s; cin >> s;
    int count = 0, ans = 0;
    for(int i=0;i<=s.size();i++) {
        if(s[i]=='A'||s[i]=='T'||s[i]=='G'||s[i]=='C') count++;
        else count = 0;
        ans = max(ans, count);
    }
    cout << ans << endl;
}
